model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -210,6 +210,12 @@
             "gemma-2b-instruct",
             "gemma-7b",
             "gemma-7b-instruct",
+            "phi-3-mini-4k-instruct",
```

Comment:
these models don't have # billion parameters written in name, can you add special casing in `recommended_hardware_config_map`?

Reply:
or provide # b parameters in `_infer_hardware`

Reply:
added parameter count for these models